Remotion/Web/Development.WebTesting/HostingStrategies/DockerHostingStrategy.cs

```diff
@@ -57,49 +83,57 @@ public class DockerHostingStrategy : IHostingStrategy
           .Distinct(StringComparer.OrdinalIgnoreCase)
           .ToArray();
 
-      var configurationParameters = new IisDockerContainerConfigurationParameters(
-          absoluteWebApplicationPath,
-          port,
-          dockerImageName,
-          dockerIsolationMode,
-          hostname,
-          is32BitProcess,
-          mounts);
+      return innerType switch
+      {
+        null => CreateIisDockerContainer(),
```

Comment:
do we need/want a default here? In particular cosnidering we might be going to switch to aspnetcore anyway in the future. Since the type is new, it's gonna be a breaking change and a required parameter, removing any ambiguity or questions.

In practice, the change wouild be harmless anyway since there's no usage yet.